usr/src/uts/common/exec/elf/elf.c

```diff
@@ -2598,8 +2628,9 @@ elfcore(vnode_t *vp, proc_t *p, cred_t *credp, rlim64_t rlimit, int sig,
 			size_t towrite = phdr[i].p_filesz;
 			size_t curoff = 0;
 
-			if (zeropg == NULL)
+			if (zeropg == NULL) {
```

Comment:
One silly question: you did this to literally make this match what I pushed into SmartOS last night?

Reply:
Yes, two variants of this file is enough and we were pretty close already.

Reply:
Thanks for the reality check.